.github/workflows/zfs-tests-sanity.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Indeed, explicit revision number gives more stability (Ubuntu 22.04 is coming soon). This is not strictly related to ASan/UBSan and since I'm touching the file, why not.

Reply:
Unrelated *.yaml file changes removed.